tests/main/interfaces-system-observe/task.yaml

```diff
@@ -10,39 +13,66 @@ details: |
 
 prepare: |
     echo "Given a snap declaring a plug on the system-observe interface is installed"
-    snapbuild $TESTSLIB/snaps/system-observe-consumer .
-    snap install --dangerous system-observe-consumer_1.0_all.snap
+    snap install --edge test-snapd-system-observe-consumer
+
+    echo "And the D-bus X11 dependencies are installed"
+    apt install -y dbus-x11
+
+    if [[ "$SPREAD_SYSTEM" != ubuntu-14.04-* ]]; then
```

Comment:
I like the suggestion but given that currently only 14.04 is affected I find more clear excluding it explicitly, we can apply it as soon as we need to exclude another system from the hostnamed test, WDYT?